integration

```diff
@@ -1161,6 +1161,7 @@
   "zakery292/monitormysolar",
   "zeronounours/HA-custom-component-energy-meter",
   "zhbjsh/homeassistant-ssh",
+  "zhheo/ha_honghui_climate",
```

Comment:
As your release notes are not in English, this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>